.github/workflows/rust.yml

```diff
@@ -17,6 +17,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --workspace --verbose
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --workspace --verbose
```

Comment:
I didn't find anything in the docs about what this option is for

Reply:
> $ cargo test --help
> ...
> --workspace               Test all packages in the workspace
> ...

Reply:
if you have multiple packages (multiple Cargo.toml files), you should use `--workspace` when running compile/test/fmt/...

Reply:
Ah I see, must have overlooked it. Thank you